src/functions/get-default-branch.ts

```diff
@@ -0,0 +1,26 @@
+import * as assert from 'assert'
+import * as core from '@actions/core'
+import {github, owner, repo} from './get-context'
+
+export async function getDefaultBranch(): Promise<string> {
+  let result: string
+  try {
+    // Get the default branch from the repo info
+    const response = await github.rest.repos.get({owner, repo})
+    result = response.data.default_branch
+    assert.ok(result, 'default_branch cannot be empty')
+  } catch (err) {
+    // Handle .wiki repo
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    if ((err as any)?.status === 404 && repo.toUpperCase().endsWith('.WIKI')) {
```

Comment:
Just curious what is this?

Reply:
i found this (can't remember where) early on in working with the release draft action when i was looking for the default branch of a repo. wiki's can be referenced like `https://github.com/user/project.wiki.git`. it looks like they don't play nice with the current [api](https://github.community/t/api-for-repositorys-wiki/902).